plugins/frontend/backstage-plugin-aws/src/components/S3BucketCard/S3BucketCard.tsx

```diff
@@ -72,11 +75,10 @@ export const S3BucketCard = (props: Props) => {
         const bucketArn = entity.metadata.annotations[S3_BUCKET_ARN_ANNOTATION];
 
         const { resource } = parseArn(bucketArn);
-        const [_, resourceId] = resource.split('/');
         const s3BucketDetails = await awsApi.getResource({
           AccountId: accountId,
           TypeName: 'AWS::S3::Bucket',
-          Identifier: resourceId,
+          Identifier: resource,
```

Comment:
This plugin was actually broken, this fixes it. It was parsing out the resource name incorrectly.